crates/gpui/src/platform/linux/x11/window.rs

```diff
@@ -557,6 +573,39 @@ impl X11Window {
             .reply()
             .unwrap()
     }
+
+    fn send_moveresize(&self, flag: u32) {
+        let state = self.0.state.borrow();
```

Comment:
```suggestion
        let state = self.0.state.borrow();
        
        self.0
            .xcb_connection
            .ungrab_pointer(x11rb::CURRENT_TIME)
            .unwrap()
            .check()
            .unwrap();
```

As per the spec *"The Client MUST release all grabs prior to sending such message (except for the _NET_WM_MOVERESIZE_CANCEL message)"*. You probably should regrab the pointer after as well, but it seems to work without doing so. This fixes resizing on native x11 gnome, which otherwise fails [here](https://gitlab.gnome.org/GNOME/mutter/-/blob/main/src/compositor/meta-window-drag.c#L1860).